index.html

```diff
@@ -1,26 +1,221 @@
+
+    <title>Ultimate Arabic front-end course 2022</title>
+  </head>
+  <body>
+    <!-- menu section -->
+    <section class="menu hidden">
+      <div class="close-btn-wrapper">
+        <button class="close-btn">
+          <img src="./imgs/close-button.svg" alt="close button" />
+        </button>
+      </div>
+      <div class="links-con">
+        <ul>
+          <li><a href="about.html">About</a></li>
+          <li><a href="#program">Program</a></li>
+          <li><a href="#join">Join</a></li>
+          <li><a href="#sponsor">Sponsor</a></li>
+          <li><a href="#news">News</a></li>
+          <li><a href="#contact">Contact</a></li>
+        </ul>
+      </div>
+    </section>
+    <!-- first nav section -->
+    <nav class="first-nav">
+      <div class="container">
+        <ul>
+          <li>
+            <a href="#"><i class="fa-brands fa-facebook-f"></i></a>
+          </li>
+          <li>
+            <a href="#"><i class="fa-brands fa-twitter"></i></a>
+          </li>
+          <li><a href="#">English</a></li>
+          <li><a href="#">My Page</a></li>
+          <li><a href="#">Logout</a></li>
+        </ul>
+      </div>
+    </nav>
+    <!-- main bar section -->
+    <nav class="navbar navbar-expand-md">
+      <div class="container">
+        <a class="navbar-brand nav-logo" href="#"
+          ><img src="imgs/Ultimate-Course.png" alt="nav-logo"
+        /></a>
+        <button class="navbar-toggler p-0">
+          <i class="fa-solid fa-bars menu-btn"></i>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarSupportedContent">
+          <ul class="navbar-nav ms-auto">
+            <li class="nav-item">
+              <a class="nav-link active" href="about.html">About</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" aria-current="page" href="#program"
+                >Program</a
+              >
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" aria-current="page" href="#">Join</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" aria-current="page" href="#sponsor"
+                >Sponsor</a
+              >
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#">News</a>
+            </li>
+          </ul>
+        </div>
+        <div class="comp-name">UC Campaign</div>
+      </div>
+    </nav>
+    <!-- header section -->
+    <section class="header">
+      <div class="container">
+        <section class="headline">
+          <h2 class="welcome">"Hello! Future Programers"</h2>
+          <h1 class="title">Ultimate Arabic front-end course 2022</h1>
+          <p class="first-p">
+            The ultimate front-end courses for arabic speakers, best Arabic
+            "speakers" in the web development that give you the tools and
+            knowledge to go further.
+          </p>
```

Comment:
- [ ] Please make sure the text is in bold letters because it gives the main information about the project. Also need to follow the original design ([See project](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015))